CHANGELOG.md

```diff
@@ -7,7 +7,13 @@ All notable changes to this project will be documented in this file.
 
 #### API breaking changes
 
-TBD
+- Removing `MaskDesignable` public methods:
+   - `maskCircle()`, use instead: `view.maskType = String(MaskType.Circle)`
```

Comment:
Hi Tom,
What's the difference between MastType.Circle.rawValue and String(MaskType.Circle)?